.pre-commit-config.yaml

```diff
@@ -0,0 +1,40 @@
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.3.0
+  hooks:
+    - id: check-ast
+    - id: debug-statements
+    - id: end-of-file-fixer
+    - id: double-quote-string-fixer
+    - id: trailing-whitespace
+
+- repo: https://gitlab.com/pycqa/flake8
+  rev: 3.7.9
+  hooks:
+    - id: flake8
+- repo: https://github.com/pre-commit/mirrors-mypy
+  rev: v0.902
+  hooks:
+    - id: mypy
+      exclude: ^tests/
+      args:
+        [--no-strict-optional, --warn-unused-ignores, --ignore-missing-imports]
```

Comment:
Don't specify args, if you wanna specify configuration for mypy please add it to `mypy.ini`